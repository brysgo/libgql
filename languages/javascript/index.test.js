const { loadGll } = require("./");


let exampleGll;
beforeAll(() => {
  exampleGll = loadGll("./examples/simple-example");
});

test("a simple graphql query", () => {

  expect(await exampleGll(`
    query TestQuery {
      currentUser {
      }
    }
  `)).toMatchSnapshot();

})
