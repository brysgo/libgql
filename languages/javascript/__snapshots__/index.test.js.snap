// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a simple graphql query 1`] = `
"{ data:
   [Object: null prototype] {
     currentUser: [Object: null prototype] { name: 'Hello World' } } }
"
`;
